src/access_nri_intake/source/builders.py

```diff
@@ -286,19 +293,18 @@ def parse_access_ncfile(cls, fname: str, time_dim: str = "time") -> tuple:
 
         Returns
         -------
-        tuple
-            A tuple containing the information parsed from the file
+        output_nc_info: AccessNCFileInfo
```

Comment:
The class is technically `_AccessNCFileInfo` (note underscore) - will not having that break things if we, e.g., auto-generate documentation?

Reply:
Good catch